common/web/types/src/main.ts

```diff
@@ -29,3 +29,5 @@ export * as KMXPlus from './kmx/kmx-plus/kmx-plus.js';
 export { UnicodeSetParser, UnicodeSet } from './ldml-keyboard/unicodeset-parser-api.js';
 export { VariableParser, MarkerParser } from './ldml-keyboard/pattern-parser.js';
 export { ElementString } from './kmx/kmx-plus/element-string.js';
+
+export * from './lexical-model-types.js';
```

Comment:
Can we enumerate all the exports here? It's busy work but it helps us be explicit about what is a supported API surface. You'll note that we do not have other top-level global exports; they are all `* as xxxx`.